packages/block-library/src/navigation-overlay-close/block.json

```diff
@@ -0,0 +1,53 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/navigation-overlay-close",
+	"title": "Navigation Overlay Close",
```

Comment:
Should this get the "__experimental": true, flag as well? It looks like most others have this field set to avoid it accidentally getting included in core?